core/src/main/dotnet/Params/TransformerArray.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+using System;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Spark.Interop;
+using Microsoft.Spark.Interop.Ipc;
+using MMLSpark.Dotnet.Wrapper;
+
+namespace Microsoft.Spark.ML.Feature.Param
+{
+    // <summary>
+    /// Array of Transformers.
+    /// </summary>
+    public class TransformerArray : IJvmObjectReferenceProvider
```

Comment:
I agree, this is used to reference the jvm under beneath, but since currently the `IJvmObjectReferenceProvider` is made public in dotnet/spark repo we need to make it public as well.

Reply:
Ah, I see. Still making the implementations [explicit](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/interfaces/explicit-interface-implementation) would stop it from showing up, which seems desirable.